src/components/pages/Gamemode/Gamemode.js

```diff
@@ -56,7 +56,6 @@ const Gamemode = ({ ...props }) => {
         : props.child.gamemode.mode === 'select' &&
           !props.child.gamemode.sp && (
             <div className="dash-container">
-              <Header />
```

Comment:
Looked around and the Header tag was unnecessarily there

Reply:
Nice catch getting rid of that doubled header